backend.go

```diff
@@ -52,15 +52,27 @@ func newBackend(backendType string) (*backendCTX, error) {
 	var ok bool
 	b := &backendCTX{}
 	log.Printf("backend type : %s", backendType)
-	if backendType == "iptables" {
+	switch backendType {
+	case "iptables":
 		tmpCtx, err := newIPTables()
 		if err != nil {
 			return nil, err
 		}
 		b.firewall, ok = tmpCtx.(backend)
 		if !ok {
-			return nil, fmt.Errorf("interface iptables is not good")
+			return nil, fmt.Errorf("interface iptables type is : %T", tmpCtx)
```

Comment:
I think we might want something more user-friendly : "Unexpected type for iptables interface : %T"